src/OctoshiftCLI.Tests/Octoshift/Commands/ReclaimMannequin/ReclaimMannequinCommandHandlerTests.cs

```diff
@@ -12,15 +12,19 @@ public class ReclaimMannequinCommandHandlerTests
 {
     private readonly Mock<OctoLogger> _mockOctoLogger = TestHelpers.CreateMock<OctoLogger>();
     private readonly Mock<ReclaimService> _mockReclaimService = TestHelpers.CreateMock<ReclaimService>();
+    private readonly ConfirmationService _confirmationService;
```

Comment:
It's probably easier (and more consistent) to mock the confirmation service in these tests.
```suggestion
    private readonly ConfirmationService _confirmationService = TestHelpers.CreateMock<ConfirmationService>();
```

Then either in the tests or the class constructor you can do something like:

```
_confirmationService.Setup(x => x.AskForConfirmation(It.IsAny<string>(), It.IsAny<string>())).Returns(true);

although there should probably be a test in here for the user saying N to the confirmation too.

Reply:
will accommodate feedback in next PR